### PR TITLE
fix: improvements to credentials-fetcher codestyle

### PR DESCRIPTION
**Pull request by saikiranakula-amzn**
*Issue #, if available:*

*Description of changes:*
Improvements to credentials-fetcher codestyle
*Testing done:*
```
[ec2-user@EC2AMAZ-Ll40tS tests]$ sudo ./gmsa_test_client 
created ticket file /var/credentials-fetcher/krbdir/6a8bccc85f67d88125c8/WebApp01
created ticket file /var/credentials-fetcher/krbdir/6a8bccc85f67d88125c8/WebApp03
Client received output for add kerberos lease: 6a8bccc85f67d88125c8
deleted ticket file /var/credentials-fetcher/krbdir/6a8bccc85f67d88125c8/WebApp01/krb5cc
deleted ticket file /var/credentials-fetcher/krbdir/6a8bccc85f67d88125c8/WebApp03/krb5cc
Client received output for delete kerberos lease: 6a8bccc85f67d88125c8
[ec2-user@EC2AMAZ-Ll40tS tests]$ sudo ./gmsa_test_client --create_kerberos_tickets_non_domain_joined sai.akula xxxx contoso.com
krb tickets will get created
created ticket file /var/credentials-fetcher/krbdir/402f6f771c8353071e4a/WebApp01
created ticket file /var/credentials-fetcher/krbdir/402f6f771c8353071e4a/WebApp03
Client received output for add kerberos lease non domain joined: 402f6f771c8353071e4a
provide a valid arg, for help use -h or --help
[ec2-user@EC2AMAZ-Ll40tS tests]$ sudo ./gmsa_test_client --renew_kerberos_tickets_non_domain_joined sai.akula xxxx contoso.com
krb tickets will get renewed
kerberos ticket renewal successful
Client received output for renew kerberos lease non domain joined
provide a valid arg, for help use -h or --help
```

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/credentials-fetcher/blob/mainline/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/credentials-fetcher/blob/mainline/CONTRIBUTING.md#commit-your-change)
- [x] I have updated any necessary documentation, including READMEs and comments (where appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have checked that my tests are not configured for a specific environment

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by smhmhmd — APPROVED** [review]

File: api/src/gmsa_service.cpp [hunk 1]
```diff
@@ -2532,7 +2531,7 @@ bool check_file_size_s3(std::string s3_arn, std::string region,
             }
             long objLen =  outcome.GetResult().GetContentLength();
             //value should be less than 2000 bytes
```
Comment (jamolina) [inline api/src/gmsa_service.cpp]: Should this comment be updated as well?

File: metadata/src/metadata.cpp [hunk 2]
```diff
@@ -136,7 +136,7 @@ int write_meta_data_json( creds_fetcher::krb_ticket_info* krb_ticket_info,
  * @return 0 or 1 for successful or failed writes
  */
 int write_meta_data_json( std::list<creds_fetcher::krb_ticket_info*> krb_ticket_info_list,
-                          std::string lease_id, std::string krb_files_dir )
+                           std::string lease_id, std::string krb_files_dir )
```
Comment (jamolina) [inline metadata/src/metadata.cpp]: Was this a typo? I think it is now misaligned with the line above.

File: auth/kerberos/src/krb.cpp [hunk 3]
```diff
@@ -1129,7 +1135,9 @@ std::string retrieve_secret_from_ecs_config(std::string ecs_variable_name)
         results = split_string(line, '=');
         std::string key = results[0];
         std::string value = results[1];
-        if ( ecs_variable_name.compare( key ) == 0 )
+        if ( !contains_invalid_characters_in_credentials(value) && ecs_variable_name.compare( key
```
Comment (jamolina) [inline auth/kerberos/src/krb.cpp]: nit: Would this be better in one line? or have the whole second clause on its own line?

File: api/src/gmsa_service.cpp (hunk 1 above)
Comment (saikiranakula-amzn) [reply api/src/gmsa_service.cpp]: addressed the comment

File: auth/kerberos/src/krb.cpp (hunk 3 above)
Comment (saikiranakula-amzn) [reply auth/kerberos/src/krb.cpp]: addressed the comment

File: metadata/src/metadata.cpp (hunk 2 above)
Comment (saikiranakula-amzn) [reply metadata/src/metadata.cpp]: addressed the comment

**Review by jamolina — APPROVED** [review]
Looks good, thank you!

**Pull request merged**